doc/releases/release-notes-4.2.rst

```diff
@@ -69,6 +69,11 @@ New APIs and options
 
     * :c:func:`bt_le_get_local_features`
 
+* Display
+
+  * Add function ``display_clear`` to the display subsystem to clear the target devices
+    screen display, resetting it to black. (:github:`12937`)
```

Comment:
just

```
:c:func:`bt_le_get_local_features`
```

the idea is that the details are already in the linked documentation, less duplication work when api gets added